src/prefect/indicators/utils.py

```diff
@@ -59,3 +76,22 @@ def get_targets_for_level(level: Level, environment: Environment) -> pd.DataFram
         raise NotImplementedError("Unsupported level %d", level)
     with Session(get_api_db_engine(environment)) as session:
         return pd.read_sql_table(level.name.lower(), con=session.connection())
+
+
+def export_indic(  # noqa: PLR0913
```

Comment:
Let's be explicit :blush: 
```suggestion
def export_indicators(  # noqa: PLR0913
```